spec/cb/models/implementations/resume/resume_spec.rb

```diff
@@ -55,7 +55,7 @@ module Models
 
         context 'and it is specifically the PerHourOrPerYear missing' do
           let(:resume_hash) do
-            { 'salaryInformation' => { 'jobTitle' => 'jerbs',
+            { 'salaryInformation' => { 'workExperienceId' => 'test1',
               'mostRecentPayAmount' => 100.00 },'userIdentifier' => 'user' }
           end
```

Comment:
@ausmith I would agree with you, but in this case when this test was written a while ago people were asking for it not to be spread out like that.  So I will ignore this unless there is enough desire from more of the team to make the switch to explicit.  Now what I will do is rubocop this file though.